Lombiq.ChartJs.csproj

```diff
@@ -39,16 +39,18 @@
 
   <ItemGroup>
     <PackageReference Include="OrchardCore.Module.Targets" Version="1.3.0" />
+    <PackageReference Include="OrchardCore.DisplayManagement" Version="1.3.0" />
+    <PackageReference Include="OrchardCore.ResourceManagement" Version="1.3.0" />
```

Comment:
Why?

Reply:
To make sure tag helpers work.

Reply:
Yeah, but iirc we agreed to put that into the OrchardCore specific HelpfulLibraries project so we don't have to include these packages in each module.

Reply:
Yep, and I did that, but this project only uses a couple of features from the `Common`, not `OrchardCore` HL project.

Reply:
I don't see the advantage, becuase this is an OC module.

Reply:
Fair enough.